README.md

```diff
@@ -206,13 +207,45 @@ To create a dynamic SCI var you can set metadata or use `sci/new-dynamic-var`:
 (sci/eval-string "(binding [*x* 12] (inc *x*))" {:namespaces {'user {'*x* x2}}}) ;;=> 13
 ```
 
-SCI vars can be bound from Clojure using `sci/binding`:
+These dynamic SCI vars can be bound from Clojure using `sci/binding`:
 
 ``` clojure
 (def x (sci/new-dynamic-var 'x 10))
 (sci/binding [x 11] (sci/eval-string "(inc *x*)" {:namespaces {'user {'*x* x}}})) ;;=> 11
 ```
 
+Notice that you cannot set _host_ dynamic variables _from your SCI scripts_ - `binding` will only work
+on dynamic variables you defined in the script itself, or on SCI dynamic variables exposed from Clojure.
+This applies also to :sci/macros (which expand into more code in the scripts). There workaround is to
+only bind them from Clojure, i.e. from functions exposed to and _called by_ your scripts:
+
+``` clojure
+(def ^:dynamic *x* 1)
+(defn with-x [x-val f] (binding [*x* x-val] (f)))
+(defn get-x [] *x*)
+(def userns (sci/create-ns 'user))
+(sci/eval-string "(with-x 42 #(get-x))"
+                 {:namespaces {'user {'with-x (sci/copy-var with-x userns)
+                                      'get-x (sci/copy-var get-x userns)}}})
+;;=> 42
+```
+
+If you want to be bind the value from your script, then you can expose a SCI dynamic var to it,
+and bind its value to the host dynamic var in Clojure:
+
+```clj
+(def ^:dynamic *x* 1)
+(def userns (sci/create-ns 'user))
+(def sci-x (sci/new-dynamic-var '*x* *x* {:ns userns}))
+(defn get-x [] (binding [*x* @sci-x] *x*)) ; bind SCI dyn var value to host var
```

Comment:
Shouldn't this read:

> ; bind host var to SCI dyn var value?

Reply:
Additional note: you can also use `(sci/copy-var ...)` to copy a dynamic var with docstring + initial value.

Reply:
You mean that instead of `sci/new-dynamic-var` I could use copy-var? How would it then know that it is dynamic? Also, https://github.com/babashka/babashka/blob/master/src/babashka/impl/pprint.clj#L38-L39 uses new-dyn.-var? Maybe I am misunderstanding what is it you suggest?

Reply:
> How would it then know that it is dynamic?

Copy var respects dynamic vars.

``` clojure
user=> (require '[sci.core :as sci])
nil
user=> (def ^:dynamic *foo* 3)
#'user/*foo*
user=> (def ctx (sci/init {:namespaces {'user {'*foo* (sci/copy-var *foo* (sci/create-ns 'user))}}}))
#'user/ctx
user=> (sci/eval-string* ctx "(binding [*foo* 3] *foo*)")
3
```

Reply:
So I could replace `(def sci-x (sci/new-dynamic-var '*x* *x* {:ns userns}))` with `(def sci-x (sci/copy-var *x* userns))` ?

Reply:
Verified it works & fixed. Thank you!